kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -427,6 +427,20 @@ object KyuubiConf {
     .timeConf
     .createWithDefault(Duration.ofMinutes(30L).toMillis)
 
+  val SESSION_CONF_IGNORE_LIST: OptionalConfigEntry[String] =
+    buildConf("session.conf.ignore.list")
+      .doc("Which key should be removed from the user's session conf.")
+      .version("1.2.0")
+      .stringConf
+      .createOptional
+
+  val SESSION_CONF_RESTRICT_LIST: OptionalConfigEntry[String] =
+    buildConf("session.conf.restrict.list")
+      .doc("Restrict the session if the user's session conf has the key.")
+      .version("1.2.0")
+      .stringConf
+      .createOptional
+
```

Comment:
I think those 2 keys should be `ConfigEntry[Seq[String]]`?

> "Restrict the session if the user's session conf has the key."

And the `restrict` in description a little bit confuse me, I think `fail` may be more easy to understand?